docs/template.rst

```diff
@@ -0,0 +1,81 @@
+.. _language-data-template:
+
+Language Data Template
+----------------------
+
+.. sourcecode:: none
+
+	two-letter language name (e.g. en):
```

Comment:
Let's refer to standard here. For instance, `two-letter language code as defined in ISO-639-1 (https://en.wikipedia.org/wiki/List_of_ISO_639-1_codes).e.g. For English, it is en`.